projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -149,7 +149,18 @@ public static function gutenblock_render_form( $atts, $content ) {
 
 		self::load_view_scripts();
 
-		return Contact_Form::parse( $atts, do_blocks( $content ) );
+		$form_html = Contact_Form::parse( $atts, do_blocks( $content ) );
+
+		/**
+		 * Custom filter to modify the rendered contact form HTML,
+		 * useful for populating hidden fields.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param string $form_html The complete form HTML.
+		 * @param array  $atts      The block attributes.
+		 */
+		return apply_filters( 'grunion_modify_rendered_contact_form_html', $form_html, $atts );
```

Comment:
I know we've used `grunion_` a lot in code (and probably filters) in the past but I thin we could move towards `jetpack_` prefix by now.

Reply:
Cool 👍 sorry for jumping early onto this, I appreciate it's in draft still :-)